java-coprocessor/src/main/java/com/example/demo/RequestHandlers/RouterRequest.java

```diff
@@ -0,0 +1,19 @@
+package com.example.demo.RequestHandlers;
+
+import com.example.demo.Models.RouterRequestBody;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.UUID;
+
+public class RouterRequest {
+    public RouterRequestBody handle(RouterRequestBody request) {
+        // In this Java sample, we are mapping the request sent by the router to a model called "RouterRequestBody"
+        // Please review that class in the "Models" folder to review what you're able to retrieve at this stage
+        // If you update the "control" property from "Continue" to something like { "break": 400 }, it will terminate the request and return the specified HTTP error
+        // See: https://www.apollographql.com/docs/router/customizations/coprocessor/
+        System.out.println(request.getControl());
```

Comment:
Is it possible to make this closer to the JS example so we have parity across languages?

E.g. output the payload and have the comment be the same in both? (you can add your additional context you already have here too but as a baseline I'd like the same comment thats over on the js side! :)

https://github.com/apollosolutions/example-coprocessor-helloworld/blob/main/coprocessor/src/index.js#L9

Reply:
The only tricky thing is that you can't log out the same response the same way you would in JS. The incoming request/response body needs to be mapped to an object first via the `@RequestBody`. It was originally getting mapped to a generic `LinkedHashMap<String, Object>` which we could log out in a similar way to the js example, but that was awful to work with so I provided a Model. The downside of that is that you'd need to use the getters for each property in the payload, let me know if that's something you'd want me to do!

Reply:
Okay, we can leave the print as is, but would love to still see the comment reflected as much as possible. :)